p2p/host.go

```diff
@@ -71,24 +59,18 @@ func NewHost(self *Peer, key libp2p_crypto.PrivKey) (Host, error) {
 	p2pHost, err := libp2p.New(ctx,
 		libp2p.ListenAddrs(listenAddr),
 		libp2p.Identity(key),
-		// libp2p.DisableRelay(),
 		libp2p.EnableNATService(),
 		libp2p.ForceReachabilityPublic(),
 	)
 	if err != nil {
 		return nil, errors.Wrapf(err, "cannot initialize libp2p host")
 	}
-	traceFile := os.Getenv("P2P_TRACEFILE")
```

Comment:
why remove tracefile option?

Reply:
It’s as Adhoc and not part of any config ,